lib/jsoncpp/json/json.h

```diff
@@ -167,15 +167,15 @@ template <typename T> class SecureAllocator {
   void deallocate(volatile pointer p, size_type n) {
     std::memset(p, 0, n * sizeof(T));
     // free using "global operator delete"
-    ::operator delete(p);
+    allocate::operator delete(p);
```

Comment:
does the comment in the line above still apply when you do this?

Reply:
yes